internal/testutil/util.go

```diff
@@ -21,15 +21,16 @@ func canUseRawSockets() error {
 	if err != nil {
 		return fmt.Errorf("getting interface: %v", err)
 	}
-	conn, err := raw.ListenPacket(intf, protoPPPoEDiscovery, &raw.Config{LinuxSockDGRAM: true})
+	conn, err := raw.ListenPacket(intf, 0x8863, &raw.Config{LinuxSockDGRAM: true})
 	if err != nil {
 		return fmt.Errorf("creating PPPoE Discovery listener: %v", err)
 	}
 	defer conn.Close()
 	return nil
 }
 
-func canTest() error {
+// CanRunPrivilegedTests returns nil if Docker and raw socket based tests can be run.
+func CanRunPrivilegedTests() error {
```

Comment:
"Can" sounds like it returns a bool.

I'd call this "Check" something? But it's not a huge deal, and it's a testutil.